base/Base.jl

```diff
@@ -128,6 +128,8 @@ include("sysinfo.jl")
 include("libc.jl")
 using .Libc: getpid, gethostname, time, memcpy, memset, memmove, memcmp
 
+const USING_STOCK_GC = occursin("stock", unsafe_string(ccall(:jl_gc_active_impl, Ptr{UInt8}, ())))
```

Comment:
Maybe we should also expose `GC.gc_active_impl()` which returns the string, and then have a separate function `GC.is_stock_gc()` that does this occursin?

Reply:
I can add `GC.gc_active_impl()`. However, I got an undefined var error when I tried adding something with `occursin` to the GC module in `gcutils.jl`. I assume due to that function only being imported later on?